spec/support/examples/a_validation_with_correct_interface.rb

```diff
@@ -0,0 +1,7 @@
+
+# let(:validation) { SomeValidator.new }
+shared_examples_for "a validation with the correct interface" do
+  it "can successfully #perform_validation" do
+    expect { validation.perform_validation }.not_to raise_error
+  end
```

Comment:
I like this validation.  Although, it's implicit in anything that calls validate.  Any call to validate will call perform_validation, and any error raised there will bork whatever test it's in.  Still, it's nice to have it split out to see that in the base good case, the validator should work.